main/src/ui/conversation_view_controller.vala

```diff
@@ -178,10 +178,12 @@ public class ConversationViewController : Object {
     private void update_conversation_topic(string? subtitle = null) {
         if (subtitle != null) {
             conversation_topic = Util.summarize_whitespaces_to_space(subtitle);
```

Comment:
Please use a dedicated variable here. `conversation_topic` is synced with the actual label in the UI, and putting raw user input into a label that has `use_markup=true` can lead to issues.

Reply:
@fiaxh done, added a `string topic_markup` at top of function